dynatrace/configuration_v1/auto_tags.py

```diff
@@ -44,295 +51,29 @@ def list(self, page_size: int = 200) -> PaginatedList["AutoTagShortRepresentatio
-    WEB_APPLICATION_TAGS = "WEB_APPLICATION_TAGS"
-    WEB_APPLICATION_TYPE = "WEB_APPLICATION_TYPE"
-    NONE = None
-
-
-class ConditionKeyType(Enum):
-    PROCESS_CUSTOM_METADATA_KEY = "PROCESS_CUSTOM_METADATA_KEY"
-    HOST_CUSTOM_METADATA_KEY = "HOST_CUSTOM_METADATA_KEY"
-    PROCESS_PREDEFINED_METADATA_KEY = "PROCESS_PREDEFINED_METADATA_KEY"
-    STRING = "STRING"
-    STATIC = "STATIC"
-    NONE = None
-
-
-class ComparisonBasicType(Enum):
-    APPLICATION_TYPE = "APPLICATION_TYPE"
-    AZURE_COMPUTE_MODE = "AZURE_COMPUTE_MODE"
-    AZURE_SKU = "AZURE_SKU"
-    BITNESS = "BITNESS"
-    CLOUD_TYPE = "CLOUD_TYPE"
-    CUSTOM_APPLICATION_TYPE = "CUSTOM_APPLICATION_TYPE"
-    DATABASE_TOPOLOGY = "DATABASE_TOPOLOGY"
-    DCRUM_DECODER_TYPE = "DCRUM_DECODER_TYPE"
-    ENTITY_ID = "ENTITY_ID"
-    HYPERVISOR_TYPE = "HYPERVISOR_TYPE"
-    INDEXED_NAME = "INDEXED_NAME"
-    INDEXED_STRING = "INDEXED_STRING"
-    INDEXED_TAG = "INDEXED_TAG"
-    INTEGER = "INTEGER"
-    IP_ADDRESS = "IP_ADDRESS"
-    MOBILE_PLATFORM = "MOBILE_PLATFORM"
-    OS_ARCHITECTURE = "OS_ARCHITECTURE"
-    OS_TYPE = "OS_TYPE"
-    PAAS_TYPE = "PAAS_TYPE"
-    SERVICE_TOPOLOGY = "SERVICE_TOPOLOGY"
-    SERVICE_TYPE = "SERVICE_TYPE"
-    SIMPLE_HOST_TECH = "SIMPLE_HOST_TECH"
-    SIMPLE_TECH = "SIMPLE_TECH"
-    STRING = "STRING"
-    SYNTHETIC_ENGINE_TYPE = "SYNTHETIC_ENGINE_TYPE"
-    TAG = "TAG"
-    NONE = None
-
-
-class AutoTagRuleType(Enum):
-    APPLICATION = "APPLICATION"
-    AWS_APPLICATION_LOAD_BALANCER = "AWS_APPLICATION_LOAD_BALANCER"
-    AWS_CLASSIC_LOAD_BALANCER = "AWS_CLASSIC_LOAD_BALANCER"
-    AWS_NETWORK_LOAD_BALANCER = "AWS_NETWORK_LOAD_BALANCER"
-    AWS_RELATIONAL_DATABASE_SERVICE = "AWS_RELATIONAL_DATABASE_SERVICE"
-    AZURE = "AZURE"
-    CUSTOM_APPLICATION = "CUSTOM_APPLICATION"
-    CUSTOM_DEVICE = "CUSTOM_DEVICE"
-    DCRUM_APPLICATION = "DCRUM_APPLICATION"
-    ESXI_HOST = "ESXI_HOST"
-    EXTERNAL_SYNTHETIC_TEST = "EXTERNAL_SYNTHETIC_TEST"
-    HOST = "HOST"
-    HTTP_CHECK = "HTTP_CHECK"
-    MOBILE_APPLICATION = "MOBILE_APPLICATION"
-    PROCESS_GROUP = "PROCESS_GROUP"
-    SERVICE = "SERVICE"
-    SYNTHETIC_TEST = "SYNTHETIC_TEST"
-    NONE = None
-
-
 class ComparisonBasic(DynatraceObject):
     def _create_from_raw_data(self, raw_element: Dict[str, Any]):
         self.operator: str = raw_element.get("operator")
         self.value: dict = raw_element.get("value")
         self.negate: bool = raw_element.get("negate")
-        self.type: ComparisonBasicType = ComparisonBasicType(raw_element.get("type"))
+        self.type: ComparisonBasicType = ComparisonBasicType(
+            raw_element.get("type"))
         self.case_sensitive: bool = raw_element.get("caseSensitive")
 
 
 class ConditionKey(DynatraceObject):
     def _create_from_raw_data(self, raw_element: Dict[str, Any]):
-        self.attribute: ConditionKeyAttribute = ConditionKeyAttribute(raw_element.get("attribute"))
```

Comment:
Were these spaces done intentionally or was it a result of the formatting step? Not wrong but doesn't seem necessary.

Reply:
missed the paragraph with the code formatting, now it should be fine regarding these newlines